pandas/core/internals/managers.py

```diff
@@ -220,16 +220,8 @@ def set_axis(self, axis: int, new_labels: Index) -> None:
 
     @property
     def _is_single_block(self) -> bool:
-        if self.ndim == 1:
-            return True
-
-        if len(self.blocks) != 1:
-            return False
-
-        blk = self.blocks[0]
-        return blk.mgr_locs.is_slice_like and blk.mgr_locs.as_slice == slice(
-            0, len(self), 1
-        )
+        # Assumes we are 2D; overriden by SingleBlockManager
+        return len(self.blocks) <= 1
```

Comment:
Is there a particular reason for this to be lte instead of eq? From the name I think the latter is more apt (and what it did previously?)

Reply:
good point, will update